src/pages/sc-cycle-counts/Forms/Controller.js

```diff
@@ -27,15 +27,15 @@ const Controller = ({ store, maxAccess, labels }) => {
     onSubmit: async () => {
       try {
```

Comment:
remove try catch

Reply:
i will remove it but don't revise this file because it is not in  this PR i make changes in it only because repository request name is changed